recipe/meta.yaml

```
@@ -38,18 +32,18 @@ outputs:
         - pip
         - setuptools
```

https://github.com/matplotlib/basemap/blob/v2.0.0/pyproject.toml#L3
```suggestion
        - setuptools >= 67.0,< 81.0
```